### PR TITLE
Add flags for skipping SSL validation and providing an alternate CA path

### PR DESCRIPTION
**Pull request by akatch**
#### General

- [x] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [x] Update README with any necessary configuration snippets

~~- [ ] Binstubs are created if needed~~

- [x] RuboCop passes

- [x] Existing tests pass

### CODE REVIEW

**Comment by majormoses** [comment]
This looks good, can I see a testing artifact? A simple gist showing redacted input and output would do.

**Comment by akatch** [comment]
You bet, I'll get one sometime this afternoon.

**Comment by majormoses** [comment]
Just waiting on that [testing artifact](https://github.com/sensu-plugins/community/blob/master/PULL_REQUEST_PROCESS.md#7-testing-artifacts)?

File: bin/check-consul-leader.rb [hunk 1]
```diff
@@ -53,6 +53,16 @@ class ConsulStatus < Sensu::Plugin::Check::CLI
          long: '--scheme SCHEME',
          default: 'http'
 
+  option :insecure,
+         description: 'set this flag to disable SSL verification',
+         short: '-k',
+         long: '--insecure'
```
Comment (majormoses) [inline bin/check-consul-leader.rb]: we should declare this as a `boolean: true` option.

File: bin/check-consul-members.rb [hunk 2]
```diff
@@ -74,12 +74,27 @@ class ConsulStatus < Sensu::Plugin::Check::CLI
          long: '--scheme SCHEME',
          default: 'http'
 
+  option :insecure,
+         description: 'if set, disables SSL verification',
+         short: '-k',
+         long: '--insecure'
```
Comment (majormoses) [inline bin/check-consul-members.rb]: we should declare this as a `boolean: true` option.

File: bin/check-consul-leader.rb [hunk 3]
```diff
@@ -77,7 +87,13 @@ def strip_ip(str)
   end
 
   def run
-    r = RestClient::Resource.new("#{config[:scheme]}://#{config[:server]}:#{config[:port]}/v1/status/leader", timeout: 5).get
+    options = { timeout: 5,
```
Comment (majormoses) [inline bin/check-consul-leader.rb]: I think we should expose the timeout as an option, someone will inevitably say that it's reasonable to wait more than 5 seconds.

File: bin/check-consul-members.rb [hunk 4]
```diff
@@ -74,12 +74,27 @@ class ConsulStatus < Sensu::Plugin::Check::CLI
...
+
+  option :capath,
+         description: 'absolute path to an alternate CA file',
+         short: '-c CAPATH',
+         long: '--capath CAPATH'
+
   def run
     url = "#{config[:scheme]}://#{config[:server]}:#{config[:port]}/v1/agent/members"
+    options = { timeout: 5,
```
Comment (majormoses) [inline bin/check-consul-members.rb]: I think we should expose the timeout as an option, someone will inevitably say that it's reasonable to wait more than 5 seconds.

File: bin/check-consul-servers.rb [hunk 5]
```diff
@@ -65,8 +65,23 @@ class ConsulStatus < Sensu::Plugin::Check::CLI
          long: '--scheme SCHEME',
          default: 'http'
 
+  option :insecure,
+         description: 'if set, disables SSL verification',
+         short: '-k',
+         long: '--insecure'
```
Comment (majormoses) [inline bin/check-consul-servers.rb]: we should declare this as a `boolean: true` option.

File: bin/check-consul-servers.rb [hunk 6]
```diff
@@ -65,8 +65,23 @@ class ConsulStatus < Sensu::Plugin::Check::CLI
...
+
+  option :capath,
+         description: 'absolute path to an alternate CA file',
+         short: '-c CAPATH',
+         long: '--capath CAPATH'
+
   def run
-    json = RestClient::Resource.new("#{config[:scheme]}://#{config[:server]}:#{config[:port]}/v1/status/peers", timeout: 5).get
+    url = "#{config[:scheme]}://#{config[:server]}:#{config[:port]}/v1/status/peers"
+    options = { timeout: 5,
```
Comment (majormoses) [inline bin/check-consul-servers.rb]: I think we should expose the timeout as an option, someone will inevitably say that it's reasonable to wait more than 5 seconds.

**Comment by akatch** [comment]
I'm really sorry, I haven't forgotten about this, I've just been really busy. Happy to make the requested changes, and will get both posted by tomorrow evening.

**Comment by akatch** [comment]
And [here](https://gist.github.com/akatch/1dc61a3be4d13a0b7a66876697813e29) is a testing artifact. Please let me know if you need anything else.

File: CHANGELOG.md [hunk 7]
```diff
@@ -4,6 +4,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+### Added
+- `check-consul-leader`, `check-consul-members`, and `check-consul-servers` now accept `--insecure` and `--capath` arguments (@akatch)
```
Comment (majormoses) [inline CHANGELOG.md]: I will update this to include a `--timeout` option.

**Comment by majormoses** [comment]
released: https://rubygems.org/gems/sensu-plugins-consul/versions/1.6.0

**Pull request merged**